docs/topics/clusterdefinitions.md

```diff
@@ -65,6 +65,7 @@ $ aks-engine get-versions
 | gcLowThreshold                   | no                        | Sets the --image-gc-low-threshold value on the kublet configuration. Default is 80. [See kubelet Garbage Collection](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | kubeletConfig                    | no                        | Configure various runtime configuration for kubelet. See `kubeletConfig` [below](#feat-kubelet-config)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| kubeReservedCgroup               | no                        | The name of a systemd slice to create for containtment of both kubelet and the container runtime. When this value is a non-empty string, a file will be dropped at `/etc/systemd/system/$KUBE_RESERVED_CGROUP.slice` creating a systemd slice. Both kubelet and docker will run in this slice. This should not point to an existing systemd slice. If this value is unspecified or specified as the empty string, kubelet and the container runtime will run in the system slice by default.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
s/containtment/containment

Reply:
Also, to be clear the default `"kubeReservedCgroup": ""` results in no functional change compared to current aks-engine:master, correct?

Reply:
> no functional change compared to current aks-engine:master, correct
Correct, that is the intent.

Will fix spelling